variables.tf

```
@@ -27,6 +27,12 @@ variable "filename" {
   description = "The path to the function's deployment package within the local filesystem"
 }
 
+variable "force_create_policy" {
+  type        = bool
+  default     = null
+  description = "Overrule whether the role policy has to be created."
```

This doesn't read nicely, can we change it to something like:

```suggestion
  description = "Force creation of IAM role policy"
```

---

This is actually copied from the `role` MCAF module, so let's use the same description for the same things (consistent) and keep it like initially added by @stefanwb